Formula/vala-language-server.rb

```diff
@@ -0,0 +1,35 @@
+class ValaLanguageServer < Formula
+  desc "Code Intelligence for Vala & Genie"
+  homepage "https://github.com/Prince781/vala-language-server"
+  url "https://github.com/Prince781/vala-language-server/archive/3e01b8383b3db3c39af276528663d6084c671455.tar.gz"
+  version "0.48.3"
+  sha256 "6f3b34bcb4e049c299ae3d5433153e4b685b0bace0ea8d761ffea266714ce841"
+  license "LGPL-2.1-only"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "vala" => :build
```

Comment:
The software links against vala so this is not just a build dependency
```suggestion
  depends_on "vala"
```